packages/mux-player/test/helpers.test.js

```diff
@@ -1,37 +1,12 @@
 import { assert } from '@open-wc/testing';
-import {
-  getSrcFromPlaybackId,
-  getPosterURLFromPlaybackId,
-  getStoryboardURLFromPlaybackId,
-  AttributeTokenList,
-} from '../src/helpers.ts';
+import { getPosterURLFromPlaybackId, getStoryboardURLFromPlaybackId, AttributeTokenList } from '../src/helpers.ts';
 
 // token key Key-Must-Be-at-least-32-bytes-in-length!
 // generated via https://dinochiesa.github.io/jwt/
 const THUMBNAIL_TOKEN = 'eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiJ0In0.BUhYQ9GTTFrrqyfT_7d2hb7FCnpmdRnzHPJulYU_yUo';
 const STORYBOARD_TOKEN = 'eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiJzIn0.aYYPU6VETdQOF04OEbhDYPOCgIxmngVBIfC6BQXbgI8';
 
 describe('helpers', () => {
-  describe('getSrcFromPlaybackId', () => {
```

Comment:
NOTE: Using playback core's `toMuxVideoURL` directly now, as it has functional parity.